helmfile/values/ingress-nginx.yaml.gotmpl

```diff
@@ -118,6 +118,23 @@ controller:
     serviceMonitor:
       enabled: true
 
+  {{- if .Values.ingressNginx.controller.chroot }}
+  image:
+    chroot: true
+  containerSecurityContext:
+    seccompProfile:
+      type: RuntimeDefault
+    capabilities:
+      drop:
+      - ALL
+      add:
+      - NET_BIND_SERVICE
```

Comment:
Does it need this if we tell it to run a non-privileged port? :thinking:

Reply:
Not entirely sure how you wanted to run it, but it seems like it is needed judging by how they talk about the capabilities on this [blog post](https://kubernetes.io/blog/2022/04/28/ingress-nginx-1-2-0/#skip-the-talk-what-do-i-need-to-use-this-new-approach) and how [ingress-nginx's upstream](https://github.com/kubernetes/ingress-nginx/blob/6c92b04edc7798dd97cd36826df4400a51047820/charts/ingress-nginx/templates/controller-psp.yaml#L84) templates look.

Reply:
Thanks for linking the sources -- it's strange, though, since it's a capability that you only need as a non-root user to bind to ports <1024. So if we don't do that (which they've made sure to leave as configurable very well!), that capability shouldn't be needed.

It does fit into the overall theme of NGINX, though: ask for more permissions than you need and then pinky-swear to not use them, I guess. :cry:

Reply:
we setup containerd to allow to bind to ports <1024 anyway so there shouldn't be any difference by having this or not

Reply:
Oh, OK. :smile: Thanks, then we can completely disregard this comment, then. :joy: